core/src/main/resources/schema/infinispan-config-14.0.xsd

```diff
@@ -452,28 +452,28 @@
     <xs:attribute name="node-name" type="xs:string">
       <xs:annotation>
         <xs:documentation>
-          Name of the current node. This is a friendly name to make logs, etc. make more sense. Defaults to a combination of host name and a random number (to differentiate multiple nodes on the same host)
+          Specifies the name of the current node. Defaults to a combination of the host name and a random number to differentiate between multiple nodes on the same host.
         </xs:documentation>
       </xs:annotation>
     </xs:attribute>
     <xs:attribute name="machine" type="xs:string">
       <xs:annotation>
         <xs:documentation>
-          The id of the machine where this node runs.
+          Specifies the ID of the machine where the node runs. Useful when multiple JVM instances run on the same node or when multiple virtual machines run on the same physical machine.
```

Comment:
Hate to be a stickler but the second sentence is incomplete. Also I find it confusing that it says "multiple JVM instances run on the same node".

For the server each instance is a dedicated JVM. So I think this might be saying that if you have multiple instances on a single machine then specifying a machine ID means that Infinispan tries to distribute nodes on different machine.

Something like:

"When a single physical machine hosts multiple JVM instances, the machine ID ensures that nodes are distributed across different physical hosts."

Also I think we can drop the bit about "multiple VMs" as it's not so relevant. What we really care about are the JVM instances.

Reply:
I think I've got a little confused here with anti-affinity. This is more about the distribution of data than it is the nodes in the cluster.

"When a single physical machine hosts multiple JVM instances, the machine ID ensures that data is distributed across different physical hosts."

I have my doubts now. I'll leave it to the experts to comment but I do think that second sentence needs more clarification.